pom.xml

```diff
@@ -356,9 +355,52 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:240min pending tasks in fixing findbugs.
+                                major bugs find in these classes are [
```

Comment:
@sebing please do not use square brackets, just type them as you would in normal sentence after commas